modules/nextflow/src/main/groovy/nextflow/processor/TaskHandler.groovy

```diff
@@ -266,7 +273,7 @@ abstract class TaskHandler {
      */
     boolean canForkProcess() {
         final max = task.processor.maxForks
-        return !max ? true : task.processor.forksCount < max
+        return !max ? true : task.processor.forksCount + getForksCount() <= max
```

Comment:
Should not be `task.processor.forksCount * getForksCount()`, I mean multiplied?

Reply:
why would it be multiplied?